Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -35,8 +37,48 @@ extension JiraService {
     }
 }
 
+// MARK: - CRP Ticket Dance
 
-// MARK: - Define a CRP Issue
+extension JiraService {
+    /// Do the CRP ticket dance, which consists of:
+    ///  - creating the CRP ticket from list of commits,
+    ///  - then create all the JIRA versions on each boards,
+    ///  - then for each board, set the Fix Version field for each ticket concerned by the CRP for that board's JIRA version
+    internal func executeCRPTicketProcess(
+        crpProjectID: JiraService.FieldType.ObjectID = .init(id: "13402"), // CRP Project
+        release: GitHubService.Release,
+        repoMapping: RepoMapping,
+        commitMessages: [String],
+        container: Request
+    ) throws -> Future<(JiraService.CreatedIssue, JiraService.FixedVersionReport)> {
```

Comment:
The code below is just the code that was [in `SlackCommand+CRP.swift` here](https://github.com/babylonhealth/Stevenson/pull/43/files#diff-5cc12f66898eff5595c3de249bbe1851L46) before, but extracted in a dedicated method.

The extraction was done for 2 purposes:

 * One is because all that logic is more related to JiraService than SlackCommand so that makes more sense to extract it here
 * It also makes it easier to debug the bot by temporarily creating a dummy route for a "crpTest" API endpoint (to which we'll be able to send requests via curl/Postman/Paw) which would configure a special `JiraService` and `crpProjectID` targeting a test board in JIRA instead of the real one, and allowing to hardcode a list of tickets to process.
   * This was very useful to debug the bot and try to reproduce issues with the CRP process while not having to post on the real CRP board nor affect real tickets from a git log but a static list of fake tickets instead.
   * This code adding that fake `crpTest` route to the `router.swift` hasn't been committed to the PR obviously since it's only to debug locally, I just keep it in a stash around